vestido/vestido-storefront/app/api/orders/[orderSlug]/cod/route.ts

```diff
@@ -0,0 +1,68 @@
+import { ZodError } from 'zod';
```

Comment:
- Deprecate dedicated COD Route.
- When paymentType is COD, insert OrderPayment Entry within CreateOrder Endpoint itself.

Reply:
Done